tracing/tracing.go

```diff
@@ -33,9 +39,32 @@ func defaultConfig() *config {
 		serviceName: envOrDefault(signalfxServiceName),
 		accessToken: envOrDefault(signalfxAccessToken),
 		url:         envOrDefault(signalfxEndpointURL),
+		globalTags:  envGlobalTags(),
+
 	}
 }
 
+// envGlobalTags extract global tags from the environment variable and parses the value in the expected format
+// key1:value1,
+func envGlobalTags() []tracer.StartOption {
+	var globalTags []tracer.StartOption
```

Comment:
This would be a good opportunity to tag the library and version by default:
`signalfx.tracing.library: go-tracing`
`signalfx.tracing.version: someNewConstantWeUpdateWithEachGitTag`.

Reply:
I'll do that in a separate PR.